ztp/gitops-subscriptions/argocd/deployment/openshift-gitops-operator.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-gitops
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-gitops-operator
+  namespace: openshift-gitops
+---
```

Comment:
I think the operator-group is not needed if we use the openshift-operators namespace for the subscription. This is where the subscription gets placed if the operator is installed from the Console UI which seems to make sense for us to follow